### PR TITLE
Using uses and collection gating

### PR DESCRIPTION
**Pull request by daagcentral**

### CODE REVIEW

File: config-overrides.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+module.exports = function override(webpackConfig) {
```
Comment (daagcentral) [inline config-overrides.js]: needed for react to accept metaplex's .mjs files

File: src/App.tsx [hunk 2]
```diff
@@ -27,15 +27,6 @@ const theme = createTheme({
   },
 });
 
-const getCandyMachineId = (): anchor.web3.PublicKey | undefined => {
```
Comment (daagcentral) [inline src/App.tsx:31]: deleting read from .env to use params on URL

**Comment by spatializes** [comment]
Got it running, hit the mint page, but couldn't get it to do a mint. Probably because I don't have the gated NFT? It would be cool to see it in action next sync up. Curious to see how you updated the contract to support uses and collection. Putting that in another repo? A forked version of the CLI?

**Review by spatializes — APPROVED** [review]

**Comment by daagcentral** [comment]
I'll have to send you the gated NFT or you can make your own. It gates based on the candy mint id passed in the URL. Use the sli, mint you ur dev wallet, go to the mint page with your candy mint id and it should work.

**Pull request merged**